config/metadata/metadata.yaml

```diff
@@ -7,3 +7,9 @@ releaseSeries:
 - major: 0
   minor: 2
   contract: v1alpha3
+- major: 0
+  minor: 3
+  contract: v1alpha3
+- major: 0
+  minor: 4
```

Comment:
I effectively skipped v0.3.0 so the version is consistent with the bootstrap provider. Not sure if that's ok..